classes/PodsUI.php

```diff
@@ -2823,383 +2823,401 @@ public function manage( $reorder = false ) {
-						<div class="alignleft actions">
-							<input type="button" value="<?php echo esc_attr( sprintf( __( 'Export all %s', 'pods' ), $this->items ) ); ?>" class="button" onclick="document.location='<?php echo $export_document_location; ?>';" />
+						<br class="clear" />
 						</div>
 						<?php
+					} else {
+						?>
+						<br class="clear" />
+						<?php
 					}//end if
-					?>
-					<br class="clear" />
-					</div>
-					<?php
-				} else {
-					?>
-					<br class="clear" />
-					<?php
-				}//end if
-			?>
-			<div class="clear"></div>
-			<?php
-			if ( empty( $this->data ) && false !== $this->default_none && false === $this->search ) {
-				?>
-				<p><?php _e( 'Please use the search filter(s) above to display data', 'pods' ); ?>
-								<?php
-								if ( $this->export ) {
 				?>
-				, <?php _e( 'or click on an Export to download a full copy of the data', 'pods' ); ?><?php } ?>.</p>
+				<div class="clear"></div>
 				<?php
-			} else {
-				$this->table( $reorder );}
-			if ( ! empty( $this->data ) ) {
-				if ( true !== $reorder && ( false !== $this->pagination_total || false !== $this->pagination ) ) {
+				if ( empty( $this->data ) && false !== $this->default_none && false === $this->search ) {
 					?>
-					<div class="tablenav">
-						<div class="tablenav-pages<?php echo esc_attr( ( $this->limit < $this->total_found || 1 < $this->page ) ? '' : ' one-page' ); ?>">
-							<?php $this->pagination( 0 ); ?>
-							<br class="clear" />
-						</div>
-					</div>
+					<p><?php _e( 'Please use the search filter(s) above to display data', 'pods' ); ?>
+									<?php
+									if ( $this->export ) {
+					?>
+					, <?php _e( 'or click on an Export to download a full copy of the data', 'pods' ); ?><?php } ?>.</p>
 					<?php
+				} else {
+					$this->table( $reorder );}
+				if ( ! empty( $this->data ) ) {
+					if ( true !== $reorder && ( false !== $this->pagination_total || false !== $this->pagination ) ) {
+						?>
+						<div class="tablenav">
+							<div class="tablenav-pages<?php echo esc_attr( ( $this->limit < $this->total_found || 1 < $this->page ) ? '' : ' one-page' ); ?>">
+								<?php $this->pagination( 0 ); ?>
+								<br class="clear" />
+							</div>
+						</div>
+						<?php
+					}
 				}
-			}
 
-			?>
-		</form>
+				?>
+			</form>
+		</div>
+		<div class="pods-admin_friends-callout_container">
+			<button class="pods-admin_friends-callout_close">
+				<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" x="0px" y="0px" viewBox="0 0 100 125" enable-background="new 0 0 100 100" xml:space="preserve"><polygon points="95,17 83,5 50,38 17,5 5,17 38,50 5,83 17,95 50,62 83,95 95,83 62,50 "/></svg>
+			</button>
+			<div class="pods-admin_friends-callout_logo-container">
+				<img class="pods-admin_friends-callout_logo" src="https://pods.io/files/2018/04/friends-of-pods-logo.svg" width="100px" alt="<?php esc_attr_e( 'Friends of Pods Logo', 'pods' ); ?>">
```

Comment:
We're going to want to add this SVG to the plugin assets directly instead of loading from our own site.